packages/headless/src/features/commerce/pagination/pagination-slice.test.ts

```diff
@@ -70,4 +74,36 @@ describe('pagination slice', () => {
 
     expect(finalState.page).toBe(0);
   });
+
+  it('sets the pagination on #fetchProductListing.fulfilled', () => {
+    const pagination = {
```

Comment:
nit, is this convention to inline these definition? Could see a use case to define a constant on top of the file.

Reply:
Done in https://github.com/coveo/ui-kit/pull/3475/commits/77429ee7d8c8f87aa6322cbbea77ff06cf3d1743